src/BuildScriptGeneratorCli/Commands/PlatformsCommand.cs

```diff
@@ -114,7 +114,7 @@ private IList<string> SortVersions(IEnumerable<string> versions)
             return result.Select(v => v.ToString()).ToList();
         }
 
-        internal override IServiceProvider GetServiceProvider(IConsole console)
+        internal IServiceProvider GetServiceProvider(IConsole console)
```

Comment:
We need to override the `TryGetServiceProvider` method here. Could you change it?

Reply:
sure .. thanks